workflow/sample_files/base-vehicle-ev-charger.xml

```diff
@@ -0,0 +1,693 @@
+                  <DuctLocation>garage</DuctLocation>
+                  <DuctSurfaceArea>50.0</DuctSurfaceArea>
+                </Ducts>
+              </AirDistribution>
+            </DistributionSystemType>
+          </HVACDistribution>
+        </HVAC>
+        <WaterHeating>
+          <WaterHeatingSystem>
+            <SystemIdentifier id='WaterHeatingSystem1'/>
+            <FuelType>electricity</FuelType>
+            <WaterHeaterType>storage water heater</WaterHeaterType>
+            <Location>garage</Location>
+            <TankVolume>40.0</TankVolume>
+            <FractionDHWLoadServed>1.0</FractionDHWLoadServed>
+            <HeatingCapacity>18767.0</HeatingCapacity>
+            <EnergyFactor>0.95</EnergyFactor>
+            <HotWaterTemperature>125.0</HotWaterTemperature>
+          </WaterHeatingSystem>
+          <HotWaterDistribution>
+            <SystemIdentifier id='HotWaterDistribution1'/>
+            <SystemType>
+              <Standard>
+                <PipingLength>50.0</PipingLength>
+              </Standard>
+            </SystemType>
+            <PipeInsulation>
+              <PipeRValue>0.0</PipeRValue>
+            </PipeInsulation>
+          </HotWaterDistribution>
+          <WaterFixture>
+            <SystemIdentifier id='WaterFixture1'/>
+            <WaterFixtureType>shower head</WaterFixtureType>
+            <LowFlow>true</LowFlow>
+          </WaterFixture>
+          <WaterFixture>
+            <SystemIdentifier id='WaterFixture2'/>
+            <WaterFixtureType>faucet</WaterFixtureType>
+            <LowFlow>false</LowFlow>
+          </WaterFixture>
+        </WaterHeating>
+        <Vehicles>
+          <Vehicle>
+            <SystemIdentifier id='Vehicle1'/>
+            <VehicleType>
+              <BatteryElectricVehicle>
+                <Battery>
+                  <NominalCapacity>
+                    <Units>kWh</Units>
+                    <Value>100.0</Value>
+                  </NominalCapacity>
+                  <UsableCapacity>
+                    <Units>kWh</Units>
+                    <Value>80.0</Value>
+                  </UsableCapacity>
+                </Battery>
+                <FractionChargedLocation>
+                  <Location>Home</Location>
+                  <Percentage>0.75</Percentage>
+                </FractionChargedLocation>
+                <ConnectedCharger idref='EVCharger1'/>
+              </BatteryElectricVehicle>
+            </VehicleType>
+            <MilesDrivenPerYear>10000.0</MilesDrivenPerYear>
+            <HoursDrivenPerWeek>14.0</HoursDrivenPerWeek>
+            <FuelEconomyCombined>
+              <Units>kWh/mile</Units>
+              <Value>0.25</Value>
+            </FuelEconomyCombined>
+          </Vehicle>
+        </Vehicles>
+        <ElectricVehicleChargers>
+          <ElectricVehicleCharger>
+            <SystemIdentifier id='EVCharger1'/>
+            <Location>garage</Location>
```

Comment:
I changed the location from garage to outside and saw no change in results. Should I have?

Reply:
I thought maybe we could see minor HVAC differences when relaxing the `is_ev` logic here https://github.com/NREL/OpenStudio-HPXML/blob/ev_batteries/HPXMLtoOpenStudio/resources/battery.rb#L98, but I don't see any changes. I am going to leave that line so if there are changes under some conditions, we could run the risk of including the discharge energy.